packages/fxa-auth-db-mysql/lib/db/mysql.js

```diff
@@ -662,23 +662,23 @@ module.exports = function (log, error) {
 
   // Delete : accountResetTokens
   // Where  : tokenId = $1
-  var DELETE_ACCOUNT_RESET_TOKEN = 'CALL deleteAccountResetToken_1(?)'
+  const DELETE_ACCOUNT_RESET_TOKEN = 'CALL deleteAccountResetToken_1(?)'
```

Comment:
You could refactor these `var` => `const` changes to make this PR slightly smaller.